project_task_default_stage/models/project.py

```diff
@@ -0,0 +1,19 @@
+# (c) 2015 Incaser Informatica S.L. - Sergio Teruel
+# (c) 2015 Incaser Informatica S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    def _get_type_common(self):
+        ids = self.env['project.task.type'].search([
+            ('case_default', '=', True)])
+        return ids
+
+    type_ids = fields.Many2many(
+        comodel_name='project.task.type', relation='project_task_type_rel',
+        column1='project_id', column2='type_id', string='Tasks Stages',
+        default=_get_type_common
```

Comment:
remove everything except for `default=_get_type_common` attribute, since everything else [was already defined](https://github.com/odoo/odoo/blob/11.0/addons/project/models/project.py#L189) in core `project` module, you have no need to define it again